Using TensorFlow backend.
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
KFold = 5 :
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3976: The name tf.nn.max_pool is deprecated. Please use tf.nn.max_pool2d instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/optimizers.py:790: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3376: The name tf.log is deprecated. Please use tf.math.log instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/ops/nn_impl.py:180: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:986: The name tf.assign_add is deprecated. Please use tf.compat.v1.assign_add instead.

2022-05-30 01:31:37.867115: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2022-05-30 01:31:37.872221: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2500005000 Hz
2022-05-30 01:31:37.872539: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55e41a2c5a00 executing computations on platform Host. Devices:
2022-05-30 01:31:37.872562: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
******************* CNNLSTM********************
confusion_matrix:
[[1628  161]
 [   0    0]]
accuracy_score: 
0.9100055897149245
precision_score: 
0.0
recall_score: 
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/sklearn/metrics/_classification.py:1248: UndefinedMetricWarning: Recall is ill-defined and being set to 0.0 due to no true samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
0.0
f1_score: 
0.0
/home/hsy/hsyb/disk_failure_prediction-master/src/utils/evaluation.py:20: RuntimeWarning: invalid value encountered in true_divide
  MCC = (TP * TN - FP * FN) / np.sqrt(hh)
MCC:  nan
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[1637  152]
 [   0    0]]
accuracy_score: 
0.9150363331470095
precision_score: 
0.0
recall_score: 
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/sklearn/metrics/_classification.py:1248: UndefinedMetricWarning: Recall is ill-defined and being set to 0.0 due to no true samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
0.0
f1_score: 
0.0
/home/hsy/hsyb/disk_failure_prediction-master/src/utils/evaluation.py:20: RuntimeWarning: invalid value encountered in true_divide
  MCC = (TP * TN - FP * FN) / np.sqrt(hh)
MCC:  nan
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[1612  156]
 [   5   16]]
accuracy_score: 
0.9100055897149245
precision_score: 
0.09302325581395349
recall_score: 
0.7619047619047619
f1_score: 
0.16580310880829016
MCC:  0.2461378534887172
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[1619  158]
 [   8    3]]
accuracy_score: 
0.9071588366890381
precision_score: 
0.018633540372670808
recall_score: 
0.2727272727272727
f1_score: 
0.03488372093023256
MCC:  0.05021233830757432
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[1611  161]
 [  10    6]]
accuracy_score: 
0.9043624161073825
precision_score: 
0.03592814371257485
recall_score: 
0.375
f1_score: 
0.06557377049180327
MCC:  0.09195553266476285
Using TensorFlow backend.
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
KFold = 5 :
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3976: The name tf.nn.max_pool is deprecated. Please use tf.nn.max_pool2d instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/optimizers.py:790: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3376: The name tf.log is deprecated. Please use tf.math.log instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/ops/nn_impl.py:180: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:986: The name tf.assign_add is deprecated. Please use tf.compat.v1.assign_add instead.

2022-05-30 08:28:14.116490: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2022-05-30 08:28:14.121556: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2500005000 Hz
2022-05-30 08:28:14.121863: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55c0c5b6d670 executing computations on platform Host. Devices:
2022-05-30 08:28:14.121886: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
******************* CNNLSTM********************
confusion_matrix:
[[781 162]
 [ 15  18]]
accuracy_score: 
0.8186475409836066
precision_score: 
0.1
recall_score: 
0.5454545454545454
f1_score: 
0.16901408450704225
MCC:  0.17414035295482705
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[801 146]
 [ 13  16]]
accuracy_score: 
0.8370901639344263
precision_score: 
0.09876543209876543
recall_score: 
0.5517241379310345
f1_score: 
0.1675392670157068
MCC:  0.18142594413015595
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[792 148]
 [ 20  16]]
accuracy_score: 
0.8278688524590164
precision_score: 
0.0975609756097561
recall_score: 
0.4444444444444444
f1_score: 
0.16
MCC:  0.1446750222753939
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[806 147]
 [ 10  12]]
accuracy_score: 
0.838974358974359
precision_score: 
0.07547169811320754
recall_score: 
0.5454545454545454
f1_score: 
0.13259668508287292
MCC:  0.1572600222164903
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[817 137]
 [ 10  11]]
accuracy_score: 
0.8492307692307692
precision_score: 
0.07432432432432433
recall_score: 
0.5238095238095238
f1_score: 
0.1301775147928994
MCC:  0.15382131097441618
Using TensorFlow backend.
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
KFold = 5 :
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3976: The name tf.nn.max_pool is deprecated. Please use tf.nn.max_pool2d instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/optimizers.py:790: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3376: The name tf.log is deprecated. Please use tf.math.log instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/ops/nn_impl.py:180: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:986: The name tf.assign_add is deprecated. Please use tf.compat.v1.assign_add instead.

2022-05-30 09:56:43.132051: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2022-05-30 09:56:43.137152: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2500005000 Hz
2022-05-30 09:56:43.137456: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5575cff30760 executing computations on platform Host. Devices:
2022-05-30 09:56:43.137481: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
******************* CNNLSTM********************
confusion_matrix:
[[222  58]
 [103 105]]
accuracy_score: 
0.6700819672131147
precision_score: 
0.6441717791411042
recall_score: 
0.5048076923076923
f1_score: 
0.5660377358490565
MCC:  0.31210644178490665
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[283  60]
 [ 49  96]]
accuracy_score: 
0.7766393442622951
precision_score: 
0.6153846153846154
recall_score: 
0.6620689655172414
f1_score: 
0.6378737541528239
MCC:  0.47737004423403046
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[270  59]
 [ 51 108]]
accuracy_score: 
0.7745901639344263
precision_score: 
0.6467065868263473
recall_score: 
0.6792452830188679
f1_score: 
0.6625766871165644
MCC:  0.4938340573182842
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[266  54]
 [ 56 112]]
accuracy_score: 
0.7745901639344263
precision_score: 
0.6746987951807228
recall_score: 
0.6666666666666666
f1_score: 
0.6706586826347305
MCC:  0.4993491485942986
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[271  61]
 [ 55 100]]
accuracy_score: 
0.7618069815195072
precision_score: 
0.6211180124223602
recall_score: 
0.6451612903225806
f1_score: 
0.6329113924050633
MCC:  0.45689409474395665
Using TensorFlow backend.
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
KFold = 5 :
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3976: The name tf.nn.max_pool is deprecated. Please use tf.nn.max_pool2d instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/optimizers.py:790: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3376: The name tf.log is deprecated. Please use tf.math.log instead.

WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/tensorflow/python/ops/nn_impl.py:180: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
WARNING:tensorflow:From /home/hsy/anaconda3/envs/tf1/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:986: The name tf.assign_add is deprecated. Please use tf.compat.v1.assign_add instead.

2022-05-30 11:33:44.951896: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2022-05-30 11:33:44.957396: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2500005000 Hz
2022-05-30 11:33:44.957731: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55ef6cd72970 executing computations on platform Host. Devices:
2022-05-30 11:33:44.957757: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
******************* CNNLSTM********************
confusion_matrix:
[[105  47]
 [ 49 125]]
accuracy_score: 
0.7055214723926381
precision_score: 
0.7267441860465116
recall_score: 
0.7183908045977011
f1_score: 
0.722543352601156
MCC:  0.40887120634546037
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[116  35]
 [ 51 123]]
accuracy_score: 
0.7353846153846154
precision_score: 
0.7784810126582279
recall_score: 
0.7068965517241379
f1_score: 
0.7409638554216867
MCC:  0.4740990593810941
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[110  33]
 [ 56 126]]
accuracy_score: 
0.7261538461538461
precision_score: 
0.7924528301886793
recall_score: 
0.6923076923076923
f1_score: 
0.7390029325513198
MCC:  0.45830965309744515
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[111  42]
 [ 45 127]]
accuracy_score: 
0.7323076923076923
precision_score: 
0.7514792899408284
recall_score: 
0.7383720930232558
f1_score: 
0.7448680351906157
MCC:  0.46343982791285815
CNNLSTM.py:31: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 4, padding="same")`
  model.add(TimeDistributed(Convolution1D(128, 4, border_mode='same'), input_shape=train_X.shape[1:]))
CNNLSTM.py:44: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  model.fit(train_X, train_y,batch_size=batch_size, nb_epoch=nb_epoch, verbose=0, validation_data=(test_X, test_y))
******************* CNNLSTM********************
confusion_matrix:
[[110  42]
 [ 60 113]]
accuracy_score: 
0.6861538461538461
precision_score: 
0.7290322580645161
recall_score: 
0.653179190751445
f1_score: 
0.6890243902439024
MCC:  0.37647704339005794
